### PR TITLE
Get AAD working again

### PR DESCRIPTION
**Pull request by fifthdegree**
Fix missing argument to token request body (probably linked to #9050). Also fix formatting issue in sdl_webview where apparently there's no way of printing a literal "%3" or "%2" without `QString::arg` replacing it.

### CODE REVIEW

File: client/SDL/sdl_webview.cpp [hunk 1]
```diff
@@ -41,6 +41,8 @@ class SchemeHandler : public QWebEngineUrlSchemeHandler
 	{
 		QUrl url = request->requestUrl();
 
+		printf("%s\n", url.query().toLocal8Bit().constData());
```
Comment (akallabeth) [inline client/SDL/sdl_webview.cpp:44]: debug code , should be removed

File: client/SDL/sdl_webview.cpp [hunk 2]
```diff
@@ -85,13 +87,13 @@ BOOL sdl_webview_get_aad_auth_code(freerdp* instance, const char* hostname, char
...
 
-	auto url = QString("https://login.microsoftonline.com/common/oauth2/v2.0/"
-	                   "authorize?client_id=%1&response_type="
-	                   "code&scope=ms-device-service%%3A%%2F%%2Ftermsrv.wvd.microsoft.com%%2Fname%%"
-	                   "2F%2%%2Fuser_impersonation&redirect_uri=%3")
-	               .arg(*client_id)
-	               .arg(hostname)
-	               .arg(*redirect_uri);
+	auto url =
+	    QString("https://login.microsoftonline.com/common/oauth2/v2.0/authorize?client_id=") +
```
Comment (akallabeth) [inline client/SDL/sdl_webview.cpp:91]: I think `\%1` or similar should also work for escaping format strings

**Review by akallabeth — CHANGES_REQUESTED** [review]
Overall LGTM

File: client/SDL/sdl_webview.cpp (hunk 2 above)
Comment (fifthdegree) [reply client/SDL/sdl_webview.cpp:91]: Tried that too, no joy

Comment (hardening) [reply client/SDL/sdl_webview.cpp:91]: Have you tried with `%%` and a single arg call (`.arg(client_id, hostname, *redirect_uri)`) ?

Comment (fifthdegree) [reply client/SDL/sdl_webview.cpp:91]: I did

**Review by hardening — APPROVED** [review]

**Comment by hardening** [comment]
@freerdp-bot can you test again please ?

File: client/SDL/sdl_webview.cpp (hunk 1 above)
Comment (akallabeth) [reply client/SDL/sdl_webview.cpp:44]: so, you gonna remove that?

**Review by hardening — APPROVED** [review]

**Comment by hardening** [comment]
@freerdp-bot test please

**Review by akallabeth — APPROVED** [review]

**Pull request merged**